sympy/physics/units/dimensions.py

```diff
@@ -590,18 +596,26 @@ def inv_can_transf_matrix(self):
         Useless method, kept for compatibility with previous versions.
 
         DO NOT USE.
-
+        
```

Comment:
Here is one of the white space instances. I have my editor set on stripping trailing white spaces.

Reply:
Can you please tell me what editor are you using so that I could use that in future.